component/build.gradle.kts

```diff
@@ -49,6 +51,10 @@ dependencies {
     implementation(libs.appcompat)
     implementation(libs.material)
 
+    implementation("com.maxkeppeler.sheets-compose-dialogs:core:1.1.1")
+    implementation("com.maxkeppeler.sheets-compose-dialogs:calendar:1.1.1")
+    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:1.1.5")
```

Comment:
Are you using these on the component module?

Reply:
@MKiperszmid Removed these. I first added the calendar but forgot to remove them.